docs/tutorials/two_screens.rst

```diff
@@ -0,0 +1,493 @@
+
+    plt.figure(figsize=(12., 8.))
+
+    plt.scatter(fd_all.to(u.mHz), tau0.to(u.us),
+                c=np.abs(brightness).value, s=5, cmap='Blues',
+                norm=LogNorm(vmin=1.e-4, vmax=1.))
+
+    plt.xlim(-50., 50.)
+    plt.ylim(0., 1300.)
+    plt.xlabel(r"differential Doppler shift $f_\mathrm{{D}}$ (mHz)")
+    plt.ylabel(r"relative geometric delay $\tau$ ($\mathrm{\mu s}$)")
+
+    cbar = plt.colorbar()
+    cbar.set_label('magnification')
+
+    plt.show()
+
+
+Making the dynamic spectrum
+===========================
+
+Define the observing frequencies and times. Make one a column vector and the
+other a row vector, so they will be broadcast against one another correctly.
+
+.. jupyter-execute::
+
+    t = np.linspace(0, 45*u.min, 300)[:, np.newaxis]
+    f = np.linspace(318.*u.MHz, 319.*u.MHz, 3000)
+
+Compute the geometric delays as a function of time, the associated geometric
+phases, then the dynamic wavefield, and finally the dynamic spectrum.
+
+.. jupyter-execute::
+
+    tau_t = (tau0[:, np.newaxis, np.newaxis]
+            + taudot[:, np.newaxis, np.newaxis] * t)
+
+    ph = phasor(f, tau_t)
+
+    dynwave = ph * brightness[:, np.newaxis, np.newaxis]
+
+    dynspec = np.abs(dynwave.sum(0))**2
+
+Plot the dynamic spectrum.
+
+.. jupyter-execute::
+
+    plt.figure(figsize=(12., 8.))
+
+    plt.imshow(dynspec.T,
+               origin='lower', aspect='auto', interpolation='none',
+               cmap='Greys', extent=axis_extent(t) + axis_extent(f), vmin=0.)
+    plt.xlabel(rf"time $t$ ({t.unit.to_string('latex')})")
+    plt.ylabel(rf"frequency $f$ ({f.unit.to_string('latex')})")
+
+    cbar = plt.colorbar()
+    cbar.set_label('normalized intensity')
+
+
+Making the secondary spectrum
+=============================
+
+Compute the conjugate spectrum, the conjugate variables, and then the secondary
+spectrum.
+
+.. jupyter-execute::
+
+    conspec = np.fft.fft2(dynspec)
+    conspec /= conspec[0, 0]
+    conspec = np.fft.fftshift(conspec)
+
+    tau = np.fft.fftshift(np.fft.fftfreq(f.size, f[1]-f[0])).to(u.us)
+    fd = np.fft.fftshift(np.fft.fftfreq(t.size, t[1]-t[0])).to(u.mHz)
+
+    secspec = np.abs(conspec)**2
+
+Plot the secondary spectrum.
+
+.. jupyter-execute::
+
+    plt.figure(figsize=(12., 8.))
+
+    plt.imshow(secspec.T.value,
+               origin='lower', aspect='auto', interpolation='none',
+               cmap='Greys', extent=axis_extent(fd) + axis_extent(tau),
+               norm=LogNorm(vmin=1.e-8, vmax=1.))
+
+    plt.xlim(-50., 50.)
+    plt.ylim(0., 1300.)
+    plt.xlabel(r"differential Doppler shift $f_\mathrm{{D}}$ "
+               rf"({fd.unit.to_string('latex')})")
+    plt.ylabel(r"relative geometric delay $\tau$ "
+               rf"({tau.unit.to_string('latex')})")
+
+    cbar = plt.colorbar()
+    cbar.set_label('normalized power')
+
+    plt.show()
+
+Overplot the arclet apices.
+
+.. jupyter-execute::
+
+    plt.figure(figsize=(12., 8.))
+
+    plt.imshow(secspec.T.value,
```

Comment:
Though `quantity_support()` seems less good with images/extents...  So, also fine to leave things as they are.